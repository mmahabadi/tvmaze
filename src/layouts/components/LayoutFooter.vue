<script setup lang="ts">
const year = new Date().getFullYear()
</script>
<template>
  <div class="footer">Copyright Â© {{ year }} TVMAZE</div>
</template>
<style scoped>
.footer {
  @apply mt-9 flex h-16 justify-center items-center bg-gray-900;
}
</style>
