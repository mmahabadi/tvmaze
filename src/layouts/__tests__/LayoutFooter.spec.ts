import { mount, type VueWrapper } from '@vue/test-utils'
import { describe, expect, beforeEach, it } from 'vitest'
import LayoutFooter from '@/layouts/components/LayoutFooter.vue'

describe('LayoutFooter', () => {
  let wrapper: VueWrapper<LayoutFooter>

  beforeEach(() => {
    wrapper = mount(LayoutFooter)
  })

  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
