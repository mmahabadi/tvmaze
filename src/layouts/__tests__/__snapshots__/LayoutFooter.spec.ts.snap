// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LayoutFooter > renders correctly 1`] = `"<div data-v-c8ff7694="" class="footer">Copyright Â© 2024 TVMAZE</div>"`;
